config/rbac/role.yaml

```diff
@@ -193,6 +193,22 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - incubator.konghq.com
```

Comment:
Is adding this new API group for faster iteration?
Are there any plans for version management of this API Group in the future? For example, will consider merging it with an existing API Group?

Reply:
Yes, it's mainly to clarify the separation - a CRD in this group is less likely to graduate. We don't want to distribute it by default - it will be a completely opt-in group. I've added a comment regarding that in the package `doc.go`: 
20a39ec